controllers/delayed.js

```diff
@@ -0,0 +1,37 @@
+'use strict';
+
+
+var redisModel = require('../models/redis'),
+    q = require('q');
+
+
+module.exports = function (app) {
+    var getDelayedModel = function(req, res){
+        var dfd = q.defer();
+        redisModel.getStatus("delayed").done(function(delayed){
```

Comment:
Good impulse to maintain code style, but when it comes at the cost of correctness… I'm concerned  that without catching errors, this UI will not be reliable. (This is true of other views in the project, of course.)

At the least I think we should file an issue about surfacing errors in the UI. And then I think it would be nice to start by surfacing errors in this view while you're in the code—less to remember later. What do you think?